scripts/start.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
```

Comment:
It seems that the linter changed all single quotes to double.
The only real change is the if statment on line 64